librz/debug/p/native/windows/windows_debug.c

```diff
@@ -9,10 +9,11 @@
 
 const DWORD wait_time = 1000;
 static RzList *lib_list = NULL;
-static PLIB_ITEM last_lib = NULL;
 
-#define w32_PROCESS_ALL_ACCESS (STANDARD_RIGHTS_REQUIRED | SYNCHRONIZE | 0xFFF)
-#define w32_THREAD_ALL_ACCESS  w32_PROCESS_ALL_ACCESS
+#define w32_PROCESS_ALL_ACCESS  (STANDARD_RIGHTS_REQUIRED | SYNCHRONIZE | 0xFFF)
```

Comment:
Everything else is fine. I just wonder if it's better to make all caps here: `W32_PROCESS_ALL_ACCESS` or something like that. Otherwise it looks weird.

Reply:
@XVilka It wasn't used anywhere anymore, so I just removed it